zetaclient/zetacore_observer.go

```diff
@@ -178,6 +178,10 @@ func (co *CoreObserver) startSendScheduler() {
 								co.logger.ZetaChainWatcher.Error().Msgf("mismatch chainid: want %d, got %d", c.ChainId, params.ReceiverChainId)
 								continue
 							}
+							if params.OutboundTxTssNonce > cctxList[0].GetCurrentOutTxParam().OutboundTxTssNonce+120 {
```

Comment:
I think we should define a constant variable for the number `120`

Reply:
[1275932](https://github.com/zeta-chain/node/pull/1370/commits/127593241e54a40750fb9b14650bebe8683791a1)